src/server/server.ts

```diff
@@ -93,6 +94,36 @@ app.post("/api/login", function(req, res) {
   })
 });
 
+app.get("/api/games", function(req,res){
+  GameModel.find({}, "-_id")
+  .then(data => {
+    res.json({data})
+  })
+  .catch(err => {
+    res.status(501).json({Error: err})
+  })
+})
+
+app.post("/api/create-game", function(req, res){
+  const {name, description, price, imgUrl, categories} = req.body;
+
+  const game = new GameModel({
+    name, 
+    description, 
+    price,
+    imgUrl,
+    categories
+  });
+
+  game.save()
+  .then(data => {
+    res.json({data});
+    console.log(data)
```

Comment:
maybe drop console.log... poor console.logs get no love

Reply:
Goodness. Thanks for checking! It's gone now